modules/authentication/src/migrations/UserIdToUserAccessTokenSchemaMigration.ts

```diff
@@ -1,17 +1,50 @@
 import ConduitGrpcSdk from '@conduitplatform/grpc-sdk';
 
-export async function UserIdToUserAccessTokenSchemaMigration(grpcSdk: ConduitGrpcSdk) {
-  const accessTokenSchemas: any[] = await grpcSdk.databaseProvider!.findMany(
-    'AccessToken',
-    { userId: { $exists: true } },
-  );
+module.exports = {
+  up: async function (grpcSdk: ConduitGrpcSdk) {
+    const database = grpcSdk.database!;
+    const schema = await database.getSchema('AccessToken');
+    const sqlTableName = schema.collectionName;
+    const dbType = await database.getDatabaseType();
+    if (dbType.type === 'MongoDB') {
+      const query = {
+        mongoQuery: {
+          updateMany: { userId: { $exists: true } },
+          options: { $rename: { userId: 'user' } },
+        },
+      };
+      await database.rawQuery('AccessToken', query);
+    } else {
+      let query = {
+        sqlQuery: {
+          query: `SELECT * FROM information_schema.columns WHERE table_name = '${sqlTableName}' AND column_name = 'userId';`,
+        },
+      };
+      const userIdExists = await database.rawQuery('AccessToken', query);
+      if (userIdExists.length === 0) return;
 
-  for (const accessTokenSchema of accessTokenSchemas) {
-    accessTokenSchema.user = accessTokenSchema.userId;
-    await grpcSdk.databaseProvider!.findByIdAndUpdate(
-      'AccessToken',
-      accessTokenSchema._id,
-      accessTokenSchema,
-    );
-  }
-}
+      query = {
+        sqlQuery: {
+          query: `SELECT * FROM "${sqlTableName}";`,
+        },
+      };
+      const accessTokens = await database.rawQuery('AccessToken', query);
+      if (accessTokens.length === 0) return;
+      query = {
+        sqlQuery: {
+          query: `ALTER TABLE "${sqlTableName}" ADD COLUMN IF NOT EXISTS "user" uuid;`,
```

Comment:
Is this required? I thought that sequelize will have already added the column(?)